src/sockets/handlers/transcribedTextHandler.js

```diff
@@ -1,20 +1,58 @@
 import { getAdKeywords } from "../../cache/adKeywordCache.js";
+import { getUserByIdService } from "../../services/users/userService.js";
 
-const transcribedTextHandler = (whisperSocket, io, userMap) => {
-  whisperSocket.off("transcribedRadioText");
+let isAdPlaying = false;
+let adEndTimers = new Map();
```

Comment:
```suggestion
const adEndTimers = new Map();
```
`adEndTimers`에 직접적인 할당을 하는게 아니라면 const로 해도 될 것 같아요!

Reply:
좋은 피드백 감사해요!
말씀해주신 것처럼 `adEndTimers` 자체에 재할당이 일어나지 않기 때문에 `const`로 선언하는 것이 좋겠네요!
또한 `userAdEndTimers`가 더 구체적으로 역할이 명확하게 드러나는 것 같아 해당 변수명으로 수정하겠습니다. ☺️